glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/Azure-Samples/network-go-manage-network-security-group
 import:
 - package: github.com/Azure/azure-sdk-for-go
-  version: ~11.0.0-beta
+  version: dev
```

Comment:
Will you remember changing this to the correct version when profiles are released?

Reply:
I'll set a reminder on my phone 😉

Reply:
But truthfully, I actually think we should probably wait to merge this PR until the changes are no longer in the `dev` branch. Then I can update this PR with `11.2.0-beta`. I only updated it to `dev` for the ease of folks trying this out in the interrim from my fork.